src/ITB/Installer.cls

```diff
@@ -54,18 +54,25 @@ ClassMethod RunWithParams(ByRef vars) As %Status
 
     try { 
         write "START INSTALLER",! 
-		
+
 		// set default values for installer variables
         for ivar="BasePath","Namespace","CreateNamespace","CreateDB","DataDBPath","CodeDBPath" {
             if '$data(vars(ivar)) set vars(ivar)=$parameter($classname(), ivar)
         }
-		$$$ThrowOnError(..RunManifest(.vars))
 
 		// check interoperability enabled on namespace
-		if '##class(%EnsembleMgr).IsEnsembleNamespace(vars("Namespace")) {
+		if ((vars("CreateNamespace") '= "yes")&&('##class(%EnsembleMgr).IsEnsembleNamespace(vars("Namespace")))) {
 			$$$ThrowStatus($$$ERROR($$$GeneralError, "Namespace "_vars("Namespace")_" has no Interoperability enabled"))
 		}
 
+		// downgrade configuration files (/config) for backwards compatibility with Ensemble (v. < 3.0)
+		if $Parameter("ITB.Info","VERSION") < 3.0 {
```

Comment:
Maybe it's better idea using $zversion or other product info instead of ITB.Info to check if it's Ensemble or IRIS

Reply:
Thanks, Yes, I am right about to upload another commit with $system.Version.GetProduct()